### ISSUE TITLE
[daemon/rpc]: L7 check endpoint without auth

### ISSUE THREAD

**Issue by casKd-dev** [transmission/transmission#1554]
I have been using healthchecks like `HEAD /transmission/web HTTP/1.1` to check if the daemon is accessible over HTTP for some time.

From commit cfce6e2e3a9b9d31a9dafedd0bdc8bf2cdb6e876 this is not possible anymore since it triggers the anti-bruteforce mechanism even if there are no authentication attempts. These should be not counted as authentication attempts since they don't even contain a authentication header.

If this is intended behaviour, is there a way to check if the web interface or rpc interface is accessible over HTTP without the need of authentication and without disabling the anti-bruteforce mechanism?

Transmission version: 3.0.0

**Comment by ckerr** [comment]
No, at least to me this wasn't the intended behavior; this is more of an unintended side-effect.
